### DISCUSSION TITLE
Suggestion: ability to choose what Folder the save games is at

### DISCUSSION THREAD

**Discussion by mariodk** [voxeltycoon/issues#519] [General]
becuase i have set my windows to have the "User" folder on a other harddrive than where my games/windows is at 
i would really like to change where the save files ends up for the game: prefer same folder as the game itself

and a game like this where i notice the filesize of the saves can get abit high : only first area unlocked and is at 81mb savefiles+3xautosaves thats 243mb of save file data i cant choose where it will save 
almost a 4th of a Gigabyte
sure not alot for people with big harddrives but might be for some people aswell as for smaller SSDs every saving every GB matters even more

**Comment by victorleblais** [comment]
I tried to change the path using a BATCH file to change the location of %appdata%, %localappdata% and/or %userprofile% but sadly, the game doesn't use those "shortcuts" to access the save folder's location.

You can however create a symbolic link between the "real folder location" and a folder of your choice.
This makes the OS (and any program) think that a folder is at one place, when it's really in another place.
It can be useful, but it can also be "dangerous": be careful when "moving" a folder with that feature.

1. Go to the folder that contains the game's user data: `%userprofile%\AppData\LocalLow\VoxelTycoon`
2. Inside that folder, you should have a folder called `VoxelTycoon` (and maybe one called `Voxel Tycoon_ Railroad to Hell 3` if you played the Halloween minigame).
3. Move that folder to the place you want your data to be saved at. For example: `D:\Games\VoxelTycoon\_AppData`
4. Now it's time to create the symbolic link
5. Open a Command Prompt in Admin mode (right click the Windows button or search for "Command Prompt" and right click > execute as administrator). It does not work in the Powershell.
6. Type the following command: `mklink /D %userprofile%\AppData\LocalLow\VoxelTycoon\VoxelTycoon D:\Games\VoxelTycoon\_AppData` (the second path is the "real location", replace it with what you want)

Now when you launch the game's .exe, you will that if you create saves, they'll be inside that second folder.

Note: to "undo" the link, just remove the file called VoxelTycoon inside your `%userprofile%\AppData\LocalLow\VoxelTycoon\` folder, it's easily identifiable since it should have an arrow on it, like shortcuts usually have.

**Comment by voxeltycoon** [comment]
You can now set the `SavesDirectory` property in `%AppData%\..\LocalLow\VoxelTycoon\VoxelTycoon\Settings.json` file to tell the game where it should look for saves instead of the default directory.
